logging/adapter_logrus.go

```diff
@@ -89,7 +90,12 @@ func Testing() *LogrusLogger {
 	return NewLogrus(logger)
 }
 
-func NewDefaultLogger(w io.Writer, debug, formatJSON bool, hooks ...logrus.Hook) *LogrusLogger {
+func NewDefaultLogger(
+	w io.Writer,
+	debug,
+	formatJSON bool,
+	otelTraces string,
```

Comment:
Why not using a simple boolean?

Reply:
I can change it if you think it makes the interface cleaner, but the related flag is a string:

https://github.com/formancehq/go-libs/pull/140/files#diff-182a7bb15c464792308889dc4b38b8465100603f2ea6f1f0b375e393fbbc6d00R35

Reply:
Updated. Please have a look now 🙇